packages/sheet/__tests__/index.js

```diff
@@ -100,18 +102,6 @@ describe('StyleSheet', () => {
     document.body.removeChild(container)
   })
 
-  it('should not throw an error when inserting a @import rule in speedy when a rule has already been inserted', () => {
```

Comment:
as this is not special-cased now this test became obsolete, I also couldn't convert it to a test which would check if we have warned correctly on non-leading imports as cssom package doesn't implement throwing for this case:
https://github.com/NV/CSSOM/blob/d600816ea9442c5e33f27ff59de536d7b7ccd239/lib/CSSStyleSheet.js#L39-L47